src/map/packets/char_update.cpp

```diff
@@ -90,8 +92,20 @@ CCharUpdatePacket::CCharUpdatePacket(CCharEntity* PChar)
     
     ref<uint8>(0x36) = flag;
 
-    if (!PChar->isDead() || PChar->m_DeathCounter == 0) //prevent this packet from resetting the homepoint timer after tractor
+    //This contains the amount of time remaining before the player is forced back to homepoint while dead
+    if (PChar->PAI->IsCurrentState<CDeathState>())
```

Comment:
PChar->isDead()?

Reply:
if a character is zoning, that means the entire ccharentity is about to get deleted and remade somewhere else (and I imagine client side it all gets overwritten again with the zone in packet too), or are you talking about zoning *in*?

Reply:
I think it happens for both zoning out and in. If I recall correctly this happens when being tractored. I can double check that though.